@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -252,6 +252,33 @@ export function isJsonExprReference(value: any): value is JsonExprReference {
     );
 }
 
+/**
+ * Extracts a StyleSet for a styleSetName from a Theme
+ *
+ * @param theme - The theme
+ * @param styleSetName - The name of the styleSet to be extracted.
+ *
+ * @return The Extracted StyleSet
+ */
+export function extractStyleSet(theme: Theme | FlatTheme, styleSetName: string): StyleSet {
```

Comment:
please mark this function with `@internal`

Reply:
removed it